package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "TableHeatmap",
-      "version": "3.4.0",
+      "version": "3.5.0",
```

Comment:
don't forget to update the version in package.json

Reply:
Thanks for the reminder! Here's the commits: [bcc655e](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/bcc655e505cf4ba5e4f5765d675fff1dce5f533a) and [77316da](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/77316da8a436197228a11f2cab20cf9bf6ce82fa)